src/views/Vaults.tsx

```diff
@@ -1,13 +1,20 @@
 import { useAtomValue } from 'jotai';
 import { appAtom, leaderAtom, networkConfigAtom } from 'store/app';
-import { useEffect } from 'react';
+import { useCallback, useEffect, useState } from 'react';
 import { watchVaultFactory } from 'service/vaults';
 import CollateralChoices from 'components/CollateralChoices';
+import ManageVaults from 'components/ManageVaults';
 import { useVaultStore } from 'store/vaults';
 
+const Modes = {
+  create: 0,
+  manage: 1,
+};
```

Comment:
[TS's enum](https://www.typescriptlang.org/docs/handbook/enums.html) would give you some helper types and reverse mapping.
```suggestion
enum Mode = {
  Create = 0,
  Manage = 1,
};
```

Reply:
Thanks, switched to an enum